src/opm/output/eclipse/AggregateAquiferData.cpp

```diff
@@ -0,0 +1,510 @@
+                using Ix = VI::SAnalyticAquifer::index;
+
+                auto cvrt = [&usys](const M unit, const double x) -> float
+                {
+                    return static_cast<float>(usys.from_si(unit, x));
+                };
+
+                // Time constant
+                const auto Tc = aquifer.C_t * aquifer.V0 / aquifer.J;
+
+                // Unit hack: *to_si()* here since we don't have a compressibility unit.
+                saaq[Ix::Compressibility] =
+                    static_cast<float>(usys.to_si(M::pressure, aquifer.C_t));
+
+                saaq[Ix::FetInitVol] = cvrt(M::liquid_surface_volume, aquifer.V0);
+                saaq[Ix::FetProdIndex] = cvrt(M::liquid_productivity_index, aquifer.J);
+                saaq[Ix::FetTimeConstant] = cvrt(M::time, Tc);
+
+                saaq[Ix::InitPressure] = cvrt(M::pressure, aquifer.p0.second);
+                saaq[Ix::DatumDepth] = cvrt(M::length, aquifer.d0);
+            }
+        } // Fetckovich
+    } // SinglePrecAnalyticAquifer
+
+    namespace SinglePrecAnalyticAquiferConn
+    {
+        std::vector<Opm::RestartIO::Helpers::WindowedArray<float>>
+        allocate(const Opm::RestartIO::InteHEAD::AquiferDims& aqDims)
+        {
+            using WA = Opm::RestartIO::Helpers::WindowedArray<float>;
+
+            return std::vector<WA>(aqDims.maxAquiferID, WA {
+                WA::NumWindows{ static_cast<WA::Idx>(aqDims.maxNumActiveAquiferConn) },
+                WA::WindowSize{ static_cast<WA::Idx>(aqDims.numRealConnElem) }
+            });
+        }
+
+        template <typename SCAqArray>
+        void staticContrib(const Opm::Aquancon::AquancCell& connection,
+                           const double                     tot_influx,
+                           SCAqArray&                       scaq)
+        {
+            using Ix = VI::SAnalyticAquiferConn::index;
+
+            auto make_ratio = [tot_influx](const double x) -> float
+            {
+                return static_cast<float>(x / tot_influx);
+            };
+
+            scaq[Ix::InfluxFraction]        = make_ratio(connection.influx_coeff);
+            scaq[Ix::FaceAreaToInfluxCoeff] = make_ratio(connection.effective_facearea);
+        }
+    } // SingleAnalyticAquiferConn
+
+    namespace DoublePrecAnalyticAquifer
+    {
+        double totalInfluxCoefficient(const Opm::UnitSystem& usys,
+                                      const double           tot_influx)
+        {
+            using M = Opm::UnitSystem::measure;
+            return usys.from_si(M::length, usys.from_si(M::length, tot_influx));
+        }
+
+        double getSummaryVariable(const Opm::SummaryState& summaryState,
+                                  const std::string&       variable,
+                                  const int                aquiferID)
+        {
+            const auto key = fmt::format("{}:{}", variable, aquiferID);
+
+            if (summaryState.has(key)) {
```

Comment:
Here you can use the overload with a default - i.e.
```C++
return summaryState.get(key, 0.0);
```

Reply:
> Here you can use the overload with a default

Thanks.  I'd forgotten about that option.

Reply:
> Here you can use the overload with a default

Thanks.  I'd forgotten about that option.